doc/DOWNLOADS.md

```diff
@@ -128,3 +118,47 @@ A stable macOS installer for Nicotine+ is available on macOS Catalina 10.15 and
 *NOTE: You have to follow [these instructions](https://support.apple.com/guide/mac-help/open-a-mac-app-from-an-unidentified-developer-mh40616/mac) the first time you open Nicotine+ on macOS.*
 
 - [Download Nicotine+ macOS Installer](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/macos-installer.zip)  [[SHA256](https://github.com/nicotine-plus/nicotine-plus/releases/latest/download/macos-installer.zip.sha256)]
+
+## Cross-Platform
+
+### pip
+
+Running Nicotine+ using pip requires a [MinGW environment](https://en.wikipedia.org/wiki/MinGW) on Windows and [Homebrew](https://brew.sh/) on macOS. Note that problems encountered using this method on OSes other than GNU/Linux variants are unlikely to receive support.
+
+To install the latest stable build of Nicotine+ using [pip](https://pip.pypa.io/), ensure the [runtime dependencies](DEPENDENCIES.md) are installed, and run the following:
+
+```sh
+pip3 install nicotine-plus
+```
+
+Keep in mind that Nicotine+ will not update automatically. When a new release is available, run the following:
+
+```sh
+pip3 install --upgrade nicotine-plus
+```
+
+## Source
```

Comment:
This section should not exist within this document.

Reply:
It can't? I thought the instructions provided did just that. They checkout a stable tag.

Reply:
Sure, perhaps it can, but the instructions provided in TESTING.md `git clone` from the master branch for the latest unstable build. To run stable you need to the correct tag to clone into the correct branch and whatnot, which is more complex.

Whilst running from Git may well be a viable solution for the technically competent, we don't want the website to make it seem to ordinary users that the project needs compiling from source before running or anything like that.